airflow/hooks/postgres_hook.py

```diff
@@ -111,3 +125,27 @@ def _serialize_cell(cell, conn):
         :rtype: object
         """
         return cell
+
+    def get_iam_token(self, conn):
+        """
+        Uses AWSHook to retrieve a temporary password to connect to Postgres
+        or Redshift. Port is required. If none is provided, default is used for
+        each service
+        """
+        redshift = conn.extra_dejson.get('redshift')
```

Comment:
Is this a new key? There's nothing pre-existing in the connection we could use is there?

Reply:
don't think there is anything in the Connection to tag it as Redshift at the moment